lib/ip_wrangler/main.rb

```diff
@@ -103,11 +99,11 @@ def check_released_resource(released_resource)
 post '/nat/port/*/*/*' do |private_ip, private_port, protocol|
   sandbox do
     private_port = private_port.to_i
-    if valid_ip? private_ip and valid_port? private_port and valid_protocol? protocol
-      public_ip_port = $nat.lock_port private_ip, private_port, protocol
+    if valid_ip?(private_ip) && valid_port?(private_port) && valid_protocol?(protocol)
+      public_ip_port = $nat.lock_port(private_ip, private_port, protocol)
 
-      if public_ip_port != nil
-        public_ip_port[:public_ip] = $config[:ext_ip]
+      if public_ip_port
+        public_ip_port[:public_ip] = $config['ext_ip']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.